lib/index.js

```diff
@@ -124,7 +124,6 @@ function draw() {
 
 function drawFrame() {
   term.color(theme.frame.color).bgColor(theme.frame.background);
-
   const topLineLength = (
```

Comment:
Same thing here. Variable assignments should be separated from function calls.
```suggestion

  const topLineLength = (
```

I noticed that I haven't followed this rule always myself, which means an automated linter really is needed here.

Reply:
Add a yml file for linter configuration. do add the rules inside it.